script.js

```diff
@@ -1,12 +1,120 @@
-//You can edit ALL of the code here
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  console.log(allEpisodes);
```

Comment:
It's usually a good iea to remove all debugging features from your PR once you finish development

Reply:
Noted,thank you very much